fs/sync/sync.go

```diff
@@ -464,6 +464,41 @@ func deleteEmptyDirectories(f fs.Fs, entries fs.DirEntries) error {
 	return nil
 }
 
+// This copies the empty directories in the slice passed in. It ignores
+// any errors copying the directories
+func copyEmptyDirectories(f fs.Fs, entries fs.DirEntries) error {
+	if len(entries) == 0 {
+		return nil
+	}
+
+	var errorCount int
+	var okCount int
+	for i := len(entries) - 1; i >= 0; i-- {
+		entry := entries[i]
+		dir, ok := entry.(fs.Directory)
+		if ok {
+			err := f.Mkdir(entry.String())
+			if err != nil {
+				fs.Debugf(fs.LogDirName(f, dir.String()), "Failed to Mkdir: %v", err)
+				errorCount++
```

Comment:
This should probably be an `Errorf `so it appears in the log and it should call `fs.Stats.Error(err)` to record the error for the exit code.